### PR TITLE
Work

### PR DESCRIPTION
**Pull request by nathancorvussolis**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16390)

### CODE REVIEW

**Review by nathancorvussolis — COMMENTED** [review]
Added version 3.0.0

File: manifests/n/nathancorvussolis/corvusskk/3.0.0/nathancorvussolis.corvusskk.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ Silent: /quiest /install /norestart
+ SilentWithProgress: /passive /install /norestart
+InstallerSuccessCodes:
+ - 3010
+Scope: machine
+UpgradeBehavior: install
+Installers:
+ - Architecture: x64
+   InstallerUrl: https://github.com/nathancorvussolis/corvusskk/releases/download/3.0.0/corvusskk-3.0.0.exe
+   InstallerSha256: 78558838C7DB038833AFBEF3E9320AD5CDB51F9F1BFCDB590FB1E792A73E05AC
```
Comment (ItzLevvie) [inline manifests/n/nathancorvussolis/corvusskk/3.0.0/nathancorvussolis.corvusskk.installer.yaml:20]:
```suggestion
```
x86 will install on both x86 and x64.

File: manifests/n/nathancorvussolis/corvusskk/3.0.0/nathancorvussolis.corvusskk.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+PackageVersion: 3.0.0
+Platform:
+ - Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: burn
+InstallModes:
+ - silent
+ - silentWithProgress
+InstallerSwitches:
+ Silent: /quiest /install /norestart
```
Comment (ItzLevvie) [inline manifests/n/nathancorvussolis/corvusskk/3.0.0/nathancorvussolis.corvusskk.installer.yaml:11]:
```suggestion
 Silent: /quiet /install /norestart
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**